index.html

```diff
@@ -206,61 +691,85 @@ <h5>Session Facilitators</h5>
+      <br> <br>
+       Please indicate your experience with the DESC collaboration and your
+       interest in the mentorship program below.
       </p>
       <ul>
-        <li> If you are new to DESC meetings and/or would like some guidance on how
-          to make the most out of this meeting, <b> would you like to be a mentee </b> and have a friendly mentor for the DESC 2021 July virtual meeting?
+
+          <div class="form-group">
+            <div>
+              <div class="form-check form-check-inline" id="inputJudo">
+                <input class="form-check-input" type="checkbox" name="judo" id="judo" value="yes">
+                <label class="form-check-label" for="judo">I'm an early-career scientist or am new to the DESC collaboration.</label>
+              </div>
+            </div>
+          </div>
+
           <div class="form-group">
             <div>
               <div class="form-check form-check-inline" id="inputMentee">
                 <input class="form-check-input" type="checkbox" name="mentee" id="mentee_yes" value="yes">
-                <label class="form-check-label" for="mentee_yes">Sign me up!</label>
+                <label class="form-check-label" for="mentee_yes">I'd like to be paired with a mentor for the DESC 2021 October Sprint Week!</label>
               </div>
             </div>
           </div>
-        </li>
 
-        <li> If you know your way around DESC and/or have attended DESC (virtual) meetings,
-          <b> would you like to be a friendly mentor</b> for the DESC 2021 July virtual meeting?
           <div class="form-group">
             <div>
               <div class="form-check form-check-inline" id="inputMentor">
                 <input class="form-check-input" type="checkbox" name="mentor" id="mentor_yes" value="yes">
-                <label class="form-check-label" for="mentor_yes">Sign me up!</label>
+                <label class="form-check-label" for="mentor_yes">I'd like to be a mentor for the DESC 2021 October Sprint Week!</label>
               </div>
             </div>
           </div>
-        </li>
 
       </ul>
+
+      <hr>
+      <h5>Tutorial Requests</h5>
+      <div class="form-group">
+      <label for="tutorial_requests">Are there any tutorials you'd like to see during the Sprint Week?</label>
+      <input type="text" class="form-control" name="tutorial_requests" id="tutorial_requests" placeholder="e.g. blending-related sessions, use of DESC catalogs" maxlength="200">
+      </div>
+
+
     <hr>
     <h5>Accommodations</h5>
-    <p>If you require any additional accommodations (e.g., closed captions,
-    alt-text for screen readers, etc), please fill out
-    <a href="https://docs.google.com/forms/d/e/1FAIpQLScdKdmaG58j4XCKEo-fr2JGwYcj7sZwMzwmeapEC6bMLnWazg/viewform">this anonymous form</a>.
+    <p>If you require any additional accommodations during the meeting, please fill out
+    <a href="https://forms.gle/eBMAa3yDrwxJqvia7">this form</a>.
```

Comment:
```suggestion
    <a href="https://forms.gle/eBMAa3yDrwxJqvia7" target="_blank">this form</a>.
```
So that we don't lose the progress when we click on this.